### PR TITLE
Bolt Site Mobile Menu Fix & Schema Table Android Scroll Fix

### PR DESCRIPTION
**Pull request by rockymountainhigh1943**
**This PR contains two important fixes:**
- Bolt Site > Removed unnecessary mobile menu item
- Schema Table > Fixed issue where schema table was not scrollable on Android

### CODE REVIEW

File: apps/bolt-site/templates/site.twig [hunk 1]
```diff
@@ -70,14 +69,6 @@
         <a href="/pattern-lab" target="_blank">Pattern Lab</a>
       {% endif %}
     {% endfor %}
-
-    <a href="#menu" class="storefront-docs__hamburger u-bolt-hide@medium">
```
Comment (sghoweri) [inline apps/bolt-site/templates/site.twig]: @rockymountainhigh1943 A for effort, but I gotta ask: have you checked out the site on your phone after making this update? 

I totally agree with removing this on smaller screens while viewing the homepage buuuut, the docs site on any other page is totally unusable as a result of the off-canvas menu getting axed:

b6766669-2c84-4c19-a003-306fb7e68e71

Comment (rockymountainhigh1943) [reply apps/bolt-site/templates/site.twig]: @sghoweri -- I absolutely tested this on my Android and both iPhoneX and iPhone8. There is no issues with mobile menus on any of these devices:

image

Can you send me an example of the issue you're talking about? I don't see any problem with any other page's navigation at mobile, tablet, or desktop.

Thanks dude!

Comment (rockymountainhigh1943) [reply apps/bolt-site/templates/site.twig]: Ok, so after looking at the doc site -- I don't see the docs navigation that was on the left being able to be triggered anymore.. whoops!! 👎

Comment (rockymountainhigh1943) [reply apps/bolt-site/templates/site.twig]: @sghoweri -- this has now been fixed and I added a few more adjustments that should really make this sing on mobile.

Check it out when you can and if it's good lets merge! :-)

**Pull request merged**